Duplicati/GUI/Duplicati.GUI.TrayIcon/HttpServerConnection.cs

```diff
@@ -495,32 +491,6 @@ public void Resume()
             ExecuteAndNotify("POST", "/serverstate/resume", req);
         }
 
-        public void StopTask(long id)

```

Comment:
The methods here are not used, but I think there is some value in leaving them here, as they make it easier to add this functionality to a tray-icon client in the future.
On the other hand, the implementation may change in the meantime leaving them with incorrect code...

Reply:
My preference would be to remove all unused code, but I see the value of leaving some things in.  If the implementation changes and these become incorrect, then either the next usage will fix them, or we will ultimately remove them.  I'll revert these removals for now.